### PR TITLE
Palak

### PR DESCRIPTION
**Pull request by Sharma-palak**
hello sir/maam please review the changes made by me in video spacing, image on home page,developer page.

### CODE REVIEW

**Comment by bvishal8510** [comment]
@Sharma-palak  never name a branch after your name, name it after the task or function implemented in it.

File: core/templates/developer.html [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+        </div>
+
+
+
+        <!--jqery, bootstrap - plugons-->
+        <script src="https://code.jquery.com/jquery-3.3.1.slim.min.js" integrity="sha384-q8i/X+965DzO0rT7abK41JStQIAqVgRVzpbzo5smXKp4YfRvH+8abtTE1Pi6jizo" crossorigin="anonymous"></script>
+        <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.14.6/umd/popper.min.js" integrity="sha384-wHAiFfRlMFy6i5SRaxvfOCifBUQy1xHdJ/yoi7FRNXMRBu5WHdZYu1hA6ZOblgut" crossorigin="anonymous"></script>
+        <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.2.1/js/bootstrap.min.js" integrity="sha384-B0UglyR+jN6CkvvICOB2joaf5I4l3gm9GU6Hc1og6Ls7i6U/mkkaduKaBhlAXv9k" crossorigin="anonymous"></script>
+
+        <!--jquery NumScroller -->
```
Comment (bvishal8510) [inline core/templates/developer.html]: these numscroller scripts are not needed.

File: core/templates/developer.html [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <title>{{ page.page_name}}</title>
+    <link rel="stylesheet" href="{%static 'css/style.css' %}">
+    <link rel="stylesheet" href="{%static 'css/animate.css' %}">
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.6.1/css/all.css" integrity="sha384-gfdkjb5BdAXd+lj+gudLWI+BXq4IuLW5IT+brZEZsLFm++aCMlF1V92rMkPaX4PP" crossorigin="anonymous">
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.2.1/css/bootstrap.min.css" integrity="sha384-GJzZqFGwb1QTTN6wy59ffF1BuGJpLSa9DkKMp0DgiMDm4iYMj70gZWKYbI706tWS" crossorigin="anonymous">
+    <link href="https://fonts.googleapis.com/css?family=Montserrat|Montserrat+Alternates|Poiret+One" rel="stylesheet">
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.6.3/css/all.css" integrity="sha384-UHRtZLI+pbxtHCWp1t77Bi1L4ZtiqrqD80Kn4Z8NTSRyMA2Fd33n5dQ8lWUE00s/" crossorigin="anonymous">
+    <script type="text/javascript" src="https://www.gstatic.com/charts/loader.js"></script>
```
Comment (bvishal8510) [inline core/templates/developer.html]: charts.js is also not required in this page

File: core/views.py [hunk 3]
```diff
@@ -296,12 +296,39 @@ def get(self, request):
...
+
+class Developer_View(View):
+    template_name = 'developer.html'
+
+    def get(self, request, *args, **kwargs):
+        display_name = "<div class='col-lg-12 mx-auto '><h3 class=' my-2'><span class='about-us'>Developer</span> </h3></div>"
+        page = Page.objects.filter(page_name='Developer').first()
+        files = Files.objects.filter(page=page)
+        images = Image.objects.filter(page=page)
+        charts = Charts.objects.filter(page=page)
```
Comment (bvishal8510) [inline core/views.py]: no charts in developers page

File: core/views.py [hunk 4]
```diff
@@ -296,12 +296,39 @@ def get(self, request):
                 data_to_frontend['message'] = 'Request failed due to internal error.'
         return JsonResponse(data_to_frontend)
 
-def view404(request):
+def view404(request,*args,**kwargs):
```
Comment (bvishal8510) [inline core/views.py]: all the error views should be at last of the page

**Comment by Sharma-palak** [comment]
ok @bvishal8510 sir

**Comment by Sharma-palak** [comment]
@bvishal8510 sir ,I have made the changes told by you .In developer.html I have removed the unnecessary scripts and removing scripts other than that would disturb nav bar.

File: core/templates/developer.html [hunk 5]
```diff
@@ -0,0 +1,39 @@
+
```
Comment (bvishal8510) [inline core/templates/developer.html]: remove these blank lines

File: core/views.py [hunk 6]
```diff
@@ -296,12 +296,26 @@ def get(self, request):
...
     error_code = 404
     error_message = 'Page Not Found'
     return render(request, 'Error.html', {'error_code':error_code, 'error_message':error_message})
 
-def view500(request):
+def view500(request,*args,**kwargs):
     error_code = 500
     error_message = 'Internal Server Error'
     return render(request, 'Error.html', {'error_code':error_code, 'error_message':error_message})
```
Comment (bvishal8510) [inline core/views.py]: According to PEP8 there should be a blank line at the end of file. So add it.

**Comment by Sharma-palak** [comment]
@bvishal8510 sir added a blank line.

**Comment by bvishal8510** [comment]
@Sharma-palak squash the commits into one commit.

**Comment by Sharma-palak** [comment]
@bvishal8510  sir ,ye sahi hai ?? After squashing commits ,when i committed it showed 
"your branch and 'origin/palak' have diverged,
and have 1 and 5 different commits each respectively.
(use 'git pull' to merge the remote branch into yours )"
so i did git pull then  git push .

**Comment by bvishal8510** [comment]
@Sharma-palak this is not correct. Follow this 
[https://github.com/todotxt/todo.txt-android/wiki/Squash-All-Commits-Related-to-a-Single-Issue-into-a-Single-Commit](https://github.com/todotxt/todo.txt-android/wiki/Squash-All-Commits-Related-to-a-Single-Issue-into-a-Single-Commit)

**Comment by Sharma-palak** [comment]
@bvishal8510  sir ,now  i am starting with pulling of code and restarting server.

**Pull request merged**